PowerValidatedSolutions.psm1

```diff
@@ -20714,6 +20721,384 @@ Function Get-PasswordPolicyConfig {
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "apiMaxFailures"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):apiMaxFailures" -value $customConfig.($product.Name).($section.Name)."apiMaxFailures" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "apiUnlockInterval"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):apiUnlockInterval" -value $customConfig.($product.Name).($section.Name)."apiUnlockInterval" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "apiRestInterval"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):apiRestInterval" -value $customConfig.($product.Name).($section.Name)."apiRestInterval" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "cliMaxFailures"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):cliMaxFailures" -value $customConfig.($product.Name).($section.Name)."cliMaxFailures" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "cliUnlockInterval"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):cliUnlockInterval" -value $customConfig.($product.Name).($section.Name)."cliUnlockInterval" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+
+    # check to see if there are any validation errors and exit script if there are errors
+    if ($encounterError -eq "True") {
+        Write-LogMessage -Type ERROR -Message "Error in validate the custom password policy file. Check the log file for details. " -Colour Red
```

Comment:
Change to:

Write-LogMessage -Type ERROR -Message "Validate Errors Found in the Password Policy Configuration File. Check the Log for Details." -Colour Red

Reply:
resolved in 2948bc273bae34312514db8aba5d7cb584efd8d4